### PR TITLE
[CELEBORN-1303] Introduce API request module for dashboard frontend

### PR DESCRIPTION
**Pull request by labbomb**
### What changes were proposed in this pull request?

- Adds api request module.
- Adds license about `@varlet/axle`.
- Adss pagination function.

### Why are the changes needed?

API request module is required to request the backend interface for dashboard frontend.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

No.

### CODE REVIEW

File: web/pnpm-lock.yaml [hunk 1]
```diff
@@ -1184,7 +1207,7 @@ packages:
     peerDependencies:
       eslint: ^6.2.0 || ^7.0.0 || ^8.0.0
       eslint-plugin-vue: ^9.0.0
-      typescript: '#'
+      typescript: '*'
```
Comment (SteNicholas) [inline web/pnpm-lock.yaml:1210]: Why change the `typescript` in this pull request?

File: web/src/api/request.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    headers: () => ({
+      Authorization: ''
+    })
+  })
+)
+
+axle.useResponseInterceptor({
+  onFulfilled(response) {
+    const { status, statusText } = response.data
+    if (status !== 200 && statusText) {
```
Comment (SteNicholas) [inline web/src/api/request.ts:42]: Does this need to log the status?

Comment (labbomb) [reply web/src/api/request.ts:42]: Maybe there is a misunderstanding of the word, this should be the returned message

File: web/pnpm-lock.yaml (hunk 1 above)
Comment (labbomb) [reply web/pnpm-lock.yaml:1210]: I didn't do anything else, this just updates automatically when the package is installed

**Review by SteNicholas — APPROVED** [review]

**Comment by SteNicholas** [comment]
LGTM. Thanks. Merged into main(v0.5.0).

**Pull request closed without merging**